app/services/albumService.js

```diff
@@ -0,0 +1,8 @@
+const errors = require('../errors'),
+  axios = require('axios');
+
+exports.getAllOfAlbumsProvider = () => {
+  return axios.get('https://jsonplaceholder.typicode.com/albums').catch(err => {
+    throw err;
```

Comment:
If you only are goind to catch it and throw it again, you can avoid this code. But anyways maybe we could log the error and then throw something more familiar like an error that indicates that the album provider is failing.